iron-overlay-manager.html

```diff
@@ -26,7 +26,38 @@
     this._minimumZ = 101;
 
     this._backdrops = [];
-  }
+
+    // Keeps track of the element to return the focus to.
+    this._lastFocusedNode = null;
+    document.body.addEventListener('focus', this._onCaptureFocus.bind(this), true);
+  };
+
+  /**
+   * @private
+   * @description Stores the last focused node if it is not contained in an overlay
+   * @param {event: Event}
+   */
+  Polymer.IronOverlayManagerClass.prototype._onCaptureFocus = function(event) {
+    var path = Polymer.dom(event).path;
+    if (!this._overlayParent(path)) {
+      this._lastFocusedNode = path[0];
+    }
+  };
+
+  /**
+   * @private
+   * @description Checks if the array contains an overlay, and returns it.
+   * @param {path: [HTMLNode]} array obtained from event.path
+   * @returns {overlat: HTMLNode|undefined} the overlay
+   */
+  Polymer.IronOverlayManagerClass.prototype._overlayParent = function(path) {
```

Comment:
`private` should be indicated with two leading `_` characters, e.g., `__overlayParent`